package/network/utils/comgt/files/ncm.json

```diff
@@ -63,5 +63,20 @@
 		},
 		"connect": "AT*ENAP=1,${profile}",
 		"disconnect": "AT*ENAP=0"
+	},
+	"mtk1": {
+		"initialize": [
+			"AT+CFUN=1"
+		],
+		"configure": [
+			"AT+CGDCONT=${profile},\\\"${pdptype}\\\",\\\"${apn}\\\",0,0",
+			"AT+CGACT=1,${profile}"
+		],
```

Comment:
Hi Matti,
here the report skipping the AT+CGACT=1,1.
The command are sent manually using picocom:

```
AT
OK
AT+CGDCONT=1,"IP","internet.wind",0,0
OK
AT+CGDATA="M-MBIM",1,1
ERROR
```